internal/service/secretsmanager/random_password_ephemeral.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package secretsmanager
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-provider-aws/internal/create"
+	"github.com/hashicorp/terraform-provider-aws/internal/framework"
+	fwflex "github.com/hashicorp/terraform-provider-aws/internal/framework/flex"
+	"github.com/hashicorp/terraform-provider-aws/names"
+)
+
+// @EphemeralResource("aws_secretsmanager_random_password", name="Random Password")
+func newEphemeralRandomPassword(context.Context) (ephemeral.EphemeralResourceWithConfigure, error) {
+	return &ephemeralRandomPassword{}, nil
+}
+
+const (
+	EPNameRandomPassword = "Random Password Ephemeral Resource"
```

Comment:
nit - Can this start with `ER` for consistency with the existing convention elsewhere?

```suggestion
	ERNameRandomPassword = "Random Password Ephemeral Resource"
```